src/routes/fetchTokenPrice.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios'
+
+const fetchTokenPrice = async(symbol) => {
+	let response = null as any
+	try {
+		response = await axios.get(`https://pro-api.coinmarketcap.com/v2/cryptocurrency/quotes/latest?symbol=${symbol}&convert=USD`, {
+			headers: {
+				'X-CMC_PRO_API_KEY': '9447d104-00b3-42db-8721-eef8e61a3332',
```

Comment:
bro you've leaked the API Key -- please:
1. remove this API Key from the repo
2. add the API Key to the env file
3. use it from the env file